setup.py

```diff
@@ -3,14 +3,27 @@
 
 # look for environment variable that specifies path to SCIP
 scipoptdir = os.environ.get('SCIPOPTDIR', '').strip('"')
+if not scipoptdir:
+    print('Environment variable SCIPOPTDIR is empty. Please specify path to SCIP.')
+    quit(1)
 
 extra_compile_args = []
 extra_link_args = []
 
 # determine include directory
 if os.path.exists(os.path.join(scipoptdir, 'src')):
-    # SCIP seems to be installed in place
-    includedir = os.path.abspath(os.path.join(scipoptdir, 'src'))
+    # SCIP seems to be installed in place; check whether it was build using make or cmake
+
+    if os.path.exists(os.path.join(scipoptdir, 'src/scip')):
+        # SCIPOPTDIR pointed to the main source directory
+        includedir = os.path.abspath(os.path.join(scipoptdir, 'src'))
+    else:
+        # SCIPOPTDIR probably pointed to a cmake build directory; try one level up
+        if os.path.exists(os.path.join(scipoptdir, '../src')):
```

Comment:
can we change this to 
```
if os.path.exists(os.path.join(scipoptdir, '../src/scip')):
```
just to be sure we are not messing with anything else?

Reply:
Yes. Fixed.